markdown_writer.py

```diff
@@ -203,6 +207,14 @@ def write_to_markdown(
                 )
             else:
                 file.write(f"| {issue.title} | {issue.html_url} |")
+            if "Assignee" in columns:
```

Comment:
Am I reading this correct, that if more than one person is assigned here it will print only the first assignee? I dont think that is ideal. I'd rather print all assignees.